lib/Chunk.js

```diff
@@ -597,4 +637,14 @@ Object.defineProperty(Chunk.prototype, "blocks", {
 	}, "Chunk.blocks: Use Chunk.addBlock/removeBlock/setBlocks to modify blocks.")
 });
 
+Object.defineProperty(Chunk.prototype, "entrypoints", {
```

Comment:
Do we want to start using `Reflect.defineProperty` instead here or for any instance? I guess the only benefit is that it has friendlier returns (which in this case is pretty relevant.

Reply:
Heh, actually check this out. https://twitter.com/bmeurer/status/928168764230848513

Reply:
Throwing is expensive, but this line doesn't throw. It's not in the hot path anyway. Not worth changing.